server/resources/TemplateResources.py

```diff
@@ -0,0 +1,22 @@
+from flask_restful import Resource
+from models.StepModel import StepModel
+from models.CampaignModel import CampaignModel
+from models.ProspectModel import ProspectModel
+from models.EmailTemplateModel import EmailTemplateModel
+from utils.RequestParserGenerator import RequestParserGenerator
+from flask_jwt_extended import jwt_required, get_jwt_identity
+
+
+class EmailTemplates(Resource):
+    @jwt_required
+    def get(self):
+        email_templates = EmailTemplateModel.get_all_templates()
```

Comment:
And then update this method once templates have an owner field

Reply:
But we want everyone to be able to have access to all the templates right?